Dockerfile

```diff
@@ -15,3 +15,8 @@ RUN echo "export JAVA_HOME=/usr/lib/jvm/java-11-openjdk" >> /etc/profile.d/jdk-e
 RUN curl -sLf -o - "https://archive.apache.org/dist/maven/maven-3/3.6.1/binaries/apache-maven-3.6.1-bin.tar.gz" | tar -C /usr/lib -zxf -
 ENV M2_HOME="/usr/lib/apache-maven-3.6.1"
 ENV PATH="${M2_HOME}/bin:${PATH}"
+
+# Install vespa-cli
+ENV VESPA_CLI_VERSION="7.469.18"
```

Comment:
I do wish we had a better mechanism for installing the latest vespa cli rather than scattering VESPA_CLI_VERSION defines all around.

Reply:
Agreed! we do auto-update versions like https://github.com/vespa-engine/sample-apps/commit/603503ae70787bb35352e4701586f92e3874bd31 , maybe we can do the same?

Reply:
Sure! I'm open to suggestions as I don't have the whole picture on all our Dockerfiles and screwdriver.yaml files.

Reply:
It would be great if we defined a source of truth for the current (most recent) Vespa version for everyone's use (i.e. all artifacts are uploaded) - maybe something like https://factory.vespa.oath.cloud/current-vespa-version that returned just the version string. Then we could use this is scripts (and in a jekyll variable for all doc).

I think factory already has this info (in the table top left) - OPENSOURCE SYSTEM version is the right one?

Reply:
I _think_ the latest commit will pick latest version